projects/elife.yaml

```diff
@@ -985,6 +986,25 @@ personalised-covers:
         ports:
             1246: 80
             1247: 8080
+
+schematron-validator:
+    domain: false
```

Comment:
it has no domain because it should only be accessible internally? `intdomain` is still there so this will acquire a `ci--schematron-validator.elife.internal` hostname only